kolibri/core/assets/src/exams/utils.js

```diff
@@ -119,6 +119,7 @@ export function fetchNodeDataAndConvertExam(exam) {
   return ContentNodeResource.fetchCollection({
     getParams: {
       ids: uniq(exam.question_sources.map(item => item.exercise_id)),
+      no_available_filtering: true,
```

Comment:
It seems like this param is newly introduced by this PR, and this is a bit of a (definitely non-blocking) nitpick but it is pretty hard for me to deduce from the name inline in some of the places it's called in the frontend what exactly this means. Does it mean the param is filtering for "none available"? Does it mean there are no available filters? I think based on the context of the PR it is filtering out unavailable content nodes, but seeing this outside of the PR, I'm not sure I would deduce that

Reply:
By default the ContentNode endpoints filter by available=True, this flag tells the backend to do no available filtering, and return all nodes. I am happy to change the name but couldn't think of a more descriptive one without being incredibly verbose.

Reply:
There probably isn't a better option! thanks for the explanation.